chord_metadata_service/patients/api_views.py

```diff
@@ -46,6 +49,68 @@ def dispatch(self, *args, **kwargs):
         return super(IndividualViewSet, self).dispatch(*args, **kwargs)
 
 
+class IndividualGetCSVViewSet(viewsets.ModelViewSet):
+    def create(self, request, *args, **kwargs):
```

Comment:
All this code is duplicated from restapi/api_renderers.py. Would it be possible, within this `create` method to initialize an IndividualCSVRenderer and call its `render` method with the data from the ORM query?

(it appears to me that the query made and the post-processing could be improved in efficiency so a refactor of this code is not out of scope. Please try to limit code duplication here)